### DISCUSSION TITLE
Is ueberzug finally dead? (prewiew-tui)

### DISCUSSION THREAD

**Discussion by amalgame21** [jarun/nnn#1684] [General]
I noticed that ueberzug is no longer maintained since the beginning of this year, but it has been working fine.
However, after the latest system update recently, the preview-tui in tmux is not functioning with the latest ueberzug 18.2.1.
I replaced `ueberzug` with `ueberzugpp` and preview-tui seems working fine now.
Is it just me to have this problem?

**Comment by N-R-K** [comment]
I think this is the issue you are facing: https://github.com/ueber-devel/ueberzug/issues/15

**Reply by amalgame21** [reply]
ya..just saw this issue, seems they just have not merge the pull request to fix it.
